config/docker-dev.yml

```diff
@@ -11,15 +11,15 @@ analysis:
       chaincontext: b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535
       from: 2550000
   storage:
-    endpoint: postgresql://indexer:password@postgres:5432/indexer?sslmode=disable
+    endpoint: postgresql://rwuser:password@postgres:5432/indexer?sslmode=disable
```

Comment:
My guess is this needs to be `@oasis-postgres` now that you've changed the container name.

FWIW, I had `indexer-postgres` in the Makefile and `local-dev.yml`. Might be worth consolidating. I was thinking `indexer-postgres` is more appropriate because the username in that postgres instance is "indexer", so it's not really intended for other oasis projects.

Reply:
> My guess is this needs to be @oasis-postgres now that you've changed the container name.

I think the url is by service name not container name.

> indexer-postgres

Hmm, I can go either way. You are suggesting we rename the user to `indexer-postgres`?

Reply:
Ah I stand corrected. If it works, it works.
Nah, I was not suggesting we rename the user. In the Makefile, I intentionally named the user `rwuser` so it's very different from variations on `indexer` -- makes for easier grepping of the username, and easier grokking of lines such as `postgresql://indexer:indexer@indexer-postgres:5432/indexer` :)

Reply:
Hmm, actually, yeah, I see the `indexer-postgres` naming now.

Reply:
I suppose then `oasis_indexer` is really just `indexer`. Uh, gonna merge first to unflake tests. Open to renaming.